llvm/lib/Transforms/Utils/LowerSwitch.cpp

```diff
@@ -569,8 +569,17 @@ class LowerSwitchLegacyPass : public FunctionPass {
     initializeLowerSwitchLegacyPassPass(*PassRegistry::getPassRegistry());
   }
 
+  // Remember the current function. Only used for verification.
+  Function *F;
+
   bool runOnFunction(Function &F) override;
 
+  void verifyAnalysis() const override {
+    for (auto &BB : *F)
+      assert(!isa<SwitchInst>(BB.getTerminator()) &&
```

Comment:
Is verifyAnalysis supposed to work in a release build? I think you're in danger of the unused member variable as is

Reply:
I'm just trying to follow existing patterns. They all seem to use `assert` so won't do anything in a normal Release build. I don't think most compilers warn about unused _public_ fields do they?